### ISSUE TITLE
nan for train and loss values when training dnn

### ISSUE THREAD

**Issue by hosseinfani** [fani-lab/OpeNTF#4]
At some point during the first epoch of dnn training on the whole dblp dataset (even with filtering), the training loss and validation loss becomes very large negative number and finally nan.

python -u main.py -data=./../raw/dblp.v12.json -domain=dblp -model=dnn -filter=1 -min_team_size=3 -min_team=3

@VaghehDashti would you please upload or put part of the log file that show the bug
